_episodes/15-lesson-study.md

```diff
@@ -181,3 +181,60 @@ Awareness of the learning process, also known as "metacognition," will also help
 >
 > This exercise and discussion should take about 15 minutes.
 {: .challenge}
+
+### Setting Up
+
+In order to communicate with learners,
+and to help us keep track of who has taught what and where,
+instructors create a website with information about their workshop.
+
+To ensure unique names for each event,
+we use the `year-month-day-venue` format for the identifier, the slug.
+For instance, for a workshop that starts on Febuary 22nd, 2022
+at the University of Data Skills,
+the slug would be `2022-02-22-data-skills`. If your workshop will be held online, you will add that to the slug as well, making it `2022-02-22-data-skills-online`.
+If you are teaching a centrally-organized workshop,
+please wait for the email from your regional coordinator
+that will tell you the slug to use for your workshop.
+
+Once your workshop website has been created,
+the host or lead instructor sends its URL to
+the [workshop coordinator](mailto:team@carpentries.org),
+who adds it to our records.
+The workshop will show up on our websites shortly thereafter.
+
+You may also create an Etherpad for your workshop
+(or other Carpentries-related event).
+When you point a browser to a url at
+`https://pad.carpentries.org/workshop-slug`
+(e.g., `https://pad.carpentries.org/2022-02-22-data-skills`
+for the example above)
+you will either arrive at an existing Etherpad by that name or a brand new
+etherpad with basic Carpentries information at the top.
```

Comment:
we should think carefully about how much of this lives in IT repo vs is a "let's look at the checklist in the handbook-- this will be updated and is a good reference..."